src/hashesorg.py

```diff
@@ -0,0 +1,26 @@
+def HashesOrg(hashed, key):

```

Comment:
I would suggest building a class of hash searchers instead of individual files, where each searcher is its own class. We have one big class which calls all the searchers combined, this will make it much easier for adding new ones. 

You can kinda see how I wanted it to work here:
https://github.com/HashPals/HashSearch/wiki/Adding-new-searchers

You build one searcher class based on a template:
https://github.com/HashPals/HashSearch/blob/main/src/searcher.py#L105

And then put it into a list
https://github.com/HashPals/HashSearch/blob/main/src/searcher.py#L14

And it loops over the list calling the `class.decrypt()` method.

My code only kinda doesn't work :)

Reply:
Agreed, Ill work on that now 👍